packages/ui-library/src/components/token-picker/index.tsx

```diff
@@ -465,7 +465,7 @@ const TokenPicker = ({
         </IconButton>
         <Grid container rowSpacing={5} direction="column" style={{ flexWrap: 'nowrap' }}>
           <Grid item xs={12} style={{ flexBasis: 'auto', alignSelf: 'flex-start' }}>
-            <Typography variant="h6" fontWeight={700}>
+            <Typography variant="h5Bold" color={({ palette }) => colors[palette.mode].typography.typo1}>
```

Comment:
double check I think this one is still used as h6

Reply:
It's h5Bold in figma (h6 is no longer supported in the new design system btw)